glue/viewers/image/qt/slice_widget.py

```diff
@@ -57,9 +58,11 @@ def sync_state_from_sliders(self, *args):
                 slices.append(self.viewer_state.slices[i])
         self.viewer_state.slices = tuple(slices)
 
+        if self.viewer_state.reference_data is not self._reference_data:
+            self._reference_data = self.viewer_state.reference_data
```

Comment:
Is the if statement really needed? Mabye just write ``self._reference_data = self.viewer_state.reference_data`` directly?

Reply:
Sure